src/styles/blocks/conteiner.scss

```diff
@@ -0,0 +1,9 @@
+.conteiner {
+  margin: 0 auto;
```

Comment:
```suggestion
```
Don't use outer spacings (margins) for the BEM blocks. `.container` is a BEM block
You could use the mix of classes instead